models/impossible_finance/bnb/impossible_finance_IDO_pools.sql

```diff
@@ -0,0 +1,104 @@
+{{
+    config(
+        alias='IDO_pools',
+        tags=['static'],
+        post_hook='{{ expose_spells(\'["bnb"]\',
+                                    "project",
+                                    "impossible_finance",
+                                    \'["kartod"]\') }}'
+    )
+}}
+
+SELECT blockchain,
+       lower(address) AS pool_address,
```

Comment:
to be safe, something i've tried to push for new static models, can we add `trim()` function here too?

Reply:
`trim()` could be used on most string columns